src/io/file_descriptor.cr

```diff
@@ -315,4 +315,18 @@ class IO::FileDescriptor < IO
   private def unbuffered_flush : Nil
     # Nothing
   end
+
+  # Blocks the current fiber until the socket is ready for read. Raises
+  # `TimeoutError` when the configured `#read_timeout` has elapsed, otherwise
+  # returns normally.
+  def wait_readable : Nil
+    system_wait_readable
+  end
+
+  # Blocks the current fiber until the socket is ready for write. Raises
+  # `TimeoutError` when the configured `#write_timeout` has elapsed, otherwise
+  # returns normally.
```

Comment:
```suggestion
  # Blocks the current fiber until the file descriptor is ready for write.
  # Raises `TimeoutError` when the configured `#write_timeout` has elapsed,
  # otherwise returns normally.
```